pkgs/applications/audio/renoise/default.nix

```diff
@@ -15,25 +14,17 @@ in
 
 stdenv.mkDerivation rec {
   pname = "renoise";
-  version = "3.1.0";
+  version = "3.2.0";
 
   src =
     if stdenv.hostPlatform.system == "x86_64-linux" then
         if releasePath == null then
-        fetchurl {
-          url = "https://files.renoise.com/demo/Renoise_${urlVersion version}_Demo_x86_64.tar.bz2";
-          sha256 = "0pan68fr22xbj7a930y29527vpry3f07q3i9ya4fp6g7aawffsga";
-        }
+		    fetchurl {
+		      url = "https://files.renoise.com/demo/Renoise_${urlVersion version}_Demo_Linux.tar.gz";
```

Comment:
I would consider changing
this to 
```
urls = [
"https://files.renoise.com/demo/Renoise_${urlVersion version}_Demo_Linux.tar.gz"
"https://web.archive.org/web/https://files.renoise.com/demo/Renoise_${urlVersion version}_Demo_Linux.tar.gz"
];
```
That way the download may work even if the original link becomes invalid.

Reply:
Good idea. Done.